### PR TITLE
Migrate CSS classes for light text 

### PR DESCRIPTION
**Pull request by ivasilov**
This PR replaces all occurences of `text-scale-1100` and `text-scale-1000` with `text-foreground-light`.

### CODE REVIEW

File: studio/components/interfaces/Organization/BillingSettingsV2/PaymentMethods/PaymentMethods.tsx [hunk 1]
```diff
@@ -19,10 +19,10 @@ import { useOrgSubscriptionQuery } from 'data/subscriptions/org-subscription-que
 import { useCheckPermissions, useStore } from 'hooks'
 import { BASE_PATH } from 'lib/constants'
 import { getURL } from 'lib/helpers'
+import Link from 'next/link'
```
Comment (MildTomato) [inline studio/components/interfaces/Organization/BillingSettingsV2/PaymentMethods/PaymentMethods.tsx:22]: ?

File: studio/components/interfaces/Database/Wrappers/WrapperRow.tsx [hunk 2]
```diff
@@ -93,27 +93,27 @@ const WrapperRow = ({
...
                       </div>
                     ))}
                     {encryptedMetadata.map((metadata) => (
                       <div key={metadata.name} className="flex items-center space-x-2 text-sm">
-                        <p className="text-scale-1000">{metadata.label}:</p>
+                        <p className="text-foreground-light">{metadata.label}:</p>
                         <Link
                           href={`/project/${ref}/settings/vault/secrets?search=${wrapper.name}_${metadata.name}`}
                         >
-                          <a className="transition text-scale-1000 hover:text-scale-1100 flex items-center space-x-2">
```
Comment (ivasilov) [inline studio/components/interfaces/Database/Wrappers/WrapperRow.tsx:108]: Needs attention.

File: studio/components/interfaces/Database/Roles/RoleRow.tsx [hunk 3]
```diff
@@ -122,7 +122,9 @@ const RoleRow = ({ role, disabled = false, onSelectDelete }: RoleRowProps) => {
                   <p
                     id="collapsible-trigger"
                     className={`text-sm ${
-                      role.active_connections > 0 ? 'text-scale-1100' : 'text-scale-1000'
```
Comment (ivasilov) [inline studio/components/interfaces/Database/Roles/RoleRow.tsx:125]: Needs attention.

File: studio/components/interfaces/Database/Roles/RoleRow.tsx [hunk 4]
```diff
@@ -122,7 +122,9 @@ const RoleRow = ({ role, disabled = false, onSelectDelete }: RoleRowProps) => {
...
+                      role.active_connections > 0
+                        ? 'text-foreground-light'
```
Comment (joshenlim) [inline studio/components/interfaces/Database/Roles/RoleRow.tsx:126]: lets do text-foreground
```suggestion
                        ? 'text-foreground'
```

File: studio/components/interfaces/Database/Wrappers/WrapperRow.tsx [hunk 5]
```diff
@@ -93,27 +93,27 @@ const WrapperRow = ({
...
+                          <a className="transition text-foreground-light hover:text-foreground-light flex items-center space-x-2">
```
Comment (joshenlim) [inline studio/components/interfaces/Database/Wrappers/WrapperRow.tsx:108]:
```suggestion
                          <a className="transition text-foreground-light hover:text-foreground flex items-center space-x-2">
```

**Review by joshenlim — APPROVED** [review]

File: studio/components/layouts/ProjectLayout/UpgradingState/UpgradingState.tsx [hunk 6]
```diff
@@ -188,8 +188,8 @@ const UpgradingState = () => {
                                 isCurrent
                                   ? 'text-foreground'
                                   : isCompleted
-                                  ? 'text-scale-1100'
```
Comment (ivasilov) [inline studio/components/layouts/ProjectLayout/UpgradingState/UpgradingState.tsx:191]: Needs attention.

**Pull request merged**